proposals/gc/Post-MVP.md

```diff
@@ -0,0 +1,563 @@
+# GC Post-v1 Extensions
+
+This document discusses various extensions which seem desirable for comprehensive support of GC types, but are intentionally left out of the [MVP](MVP.md), in order to keep its scope manageable.
```

Comment:
Perhaps add a sentence to the effect of: "Over the course of implementing and validating the MVP, it is possible that features in this list may be promoted to the MVP if experience shows that MVP performance would not otherwise be viable." ?

Reply:
Done.